akvo/rsr/views/py_reports.py

```diff
@@ -31,3 +40,222 @@ def check(request):
+        Coordinate(p.primary_location.latitude, p.primary_location.longitude)
+        for p
+        in projects
+    ]
+
+    html = render_to_string(
+        'reports/organisation-projects-results-indicators-map-overview.html',
+        context={
+            'title': 'Results and indicators overview for projects in {}'.format(country.name),
+            'staticmap': get_staticmap_url(coordinates, Size(945, 630)),
+            'projects': [
+                {'title': p.title, 'results': _transform_project_results(p, start_date, end_date)}
+                for p
+                in projects
+            ],
+            'show_comment': show_comment,
+        }
+    )
+
+    if request.GET.get('show-html', ''):
+        return HttpResponse(html)
+
+    now = datetime.today()
+
+    return _make_pdf_response(
+        html,
+        options={
+            'page-size': 'A4',
+            'orientation': 'Portrait',
+            'margin-left': '0.28in',
+            'margin-top': '1in',
+            'margin-right': '0.28in',
+            'margin-bottom': '1in',
+            'footer-left': 'Akvo RSR Report {}'.format(now.strftime('%d-%b-%Y')),
+            'footer-right': '[page]',
+            'footer-font-size': 8,
+            'footer-font-name': 'Roboto Condensed',
+            'footer-spacing': 10,
+        },
+        filename='{}-{}-{}-projects-results-indicators-overview.pdf'.format(
+            now.strftime('%Y%b%d'), organisation.id, country.iso_code
+        )
+    )
+
+
+@login_required
+def render_project_results_indicators_map_overview(request, project_id):
+    show_comment = True if request.GET.get('comment', '').strip() == 'true' else False
+    start_date = _parse_date(request.GET.get('start_date', '').strip(), datetime(1900, 1, 1))
+    end_date = _parse_date(request.GET.get('end_date', '').strip(), datetime(2999, 12, 31))
+
+    project = get_object_or_404(
+        Project.objects.prefetch_related(
+            'partners',
+            'related_projects',
+            'related_to_projects',
+            'results',
+            'results__indicators',
+            'results__indicators__periods'
+        ),
+        pk=project_id
+    )
+    location = project.primary_location
+
+    html = render_to_string(
+        'reports/project-results-indicators-map-overview.html',
+        context={
+            'project': project,
+            'location': ", ".join(filter(None, [location.city, getattr(location.country, 'name', None)])),
+            'staticmap': get_staticmap_url([Coordinate(location.latitude, location.longitude)], Size(945, 630)),
+            'results': _transform_project_results(project, start_date, end_date),
+            'show_comment': show_comment,
+        }
+    )
+
+    print(request.GET, show_comment, start_date, end_date)
+
+    if request.GET.get('show-html', ''):
+        return HttpResponse(html)
+
+    now = datetime.today()
+
+    return _make_pdf_response(
+        html,
+        options={
```

Comment:
We only have a couple of reports, so this may be fine. But, if most of this configuration seems pretty standard to me, and may be we can make it a default that we over ride.